spec/services/draw_activator_spec.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+require 'spec_helper'
```

Comment:
notably our `spec_helper` and `rails_helper` configurations are different -- I've been using `rails_helper` everywhere because of the DatabaseCleaner config